src/Reader/XLSX/Helper/DateFormatHelper.php

```diff
@@ -78,11 +78,19 @@ public static function toPHPDateFormat(string $excelDateFormat): string
                 continue;
             }
 
-            // Make sure all characters are lowercase, as the mapping table is using lowercase characters
-            $transformedPart = strtolower($dateFormatPart);
+            // Do not set \T to lower case or remove the slash
+            $dateFormatPartSections = explode('\T', $dateFormatPart);
+            foreach($dateFormatPartSections as &$dateFormatPartSection) {
 
-            // Remove escapes related to non-format characters
-            $transformedPart = str_replace('\\', '', $transformedPart);
+                // Make sure all characters are lowercase, as the mapping table is using lowercase characters
+                $dateFormatPartSection = strtolower($dateFormatPartSection);
+
+                // Remove escapes related to non-format characters
+                $dateFormatPartSection = str_replace('\\', '', $dateFormatPartSection);
```

Comment:
Infection says this is not tested, may I ask you why you wrote this line of code?